_inc/jetpack-jitm.js

```diff
@@ -176,6 +180,21 @@ jQuery( document ).ready( function( $ ) {
 				}, 2000 );
 			} );
 		} );
+
+		// Handle CTA click events.
+		$template.find( '.jitm-button[data-click][data-module="tos"]' ).click( function( e ) {
```

Comment:
Could this be more generalized, so that we don't need to update this file every time a JITM wants to have CTA click events? Maybe `CTA.click_handler` is a slug (like `tos`), and the callback is defined elsewhere? I dunno, just trying to imagine how we can keep as much of a specific JITM defined server-side as possible or, if nothing else, kept out of the general JITM plumbing.

Reply:
Since the callback is a client function, we need to define it on Jetpack.

I tried to keep as much as possible on the WPCOM side, but the only way I could think of having the acceptance code there is to avoid any client logic and perform server redirects with `jetpack.com/redirect` which is also a bit awful. 

I'll try to move the callback to the new Jetpack ToS module.

Reply:
Let me know if it looks better after last update.

Rather than having specific click handlers, JITMs will trigger AJAX actions if an action is given in `CTA.ajax_action`. The code will always be the same for all AJAX actions (trigger AJAX request and hide the JITM on successful responses), but we still need to add the AJAX endpoint in Jetpack (but at least this code is out of the JITM logic).